### PR TITLE
Add World Bank example

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
## Purpose
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/2338
## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

**Review by BuddhiWathsala — APPROVED** [review]

File: examples/world-bank/worldbank.bal [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    json j1 = check xmldata:toJson(resp);
+    io:println(j1);
+
+    // Converts the xml value to a json value with a custom attribute prefix.
+    json j2 = check xmldata:toJson(resp, options = {attributePrefix: "$"});
+    io:println(j2);
+
+    // Converts the xml value to a user-defined record type.
+    io:println("output as a WorldBank record :");
+    WorldBank w = check xmldata:toRecord(resp, false);
```
Comment (daneshk) [inline examples/world-bank/worldbank.bal:54]:
```suggestion
    WorldBank w = check xmldata:toRecord(resp, preserveNamespaces  = false);
```

Comment (daneshk) [reply examples/world-bank/worldbank.bal:54]: Shall we add a named parameter?

**Review by daneshk — DISMISSED** [review]
Add one suggestion. Overall LGTM

File: examples/world-bank/worldbank.bal (hunk 1 above)
Comment (MadhukaHarith92) [reply examples/world-bank/worldbank.bal]: Updated in 6e713ae

**Review by daneshk — APPROVED** [review]

**Pull request merged**